CountItems/CountWorker.js

```diff
@@ -35,7 +36,10 @@ class CountWorker {
         return async.waterfall([
             next => this.client._getIsTransient(bucketInfo, this.log, next),
             (isTransient, next) => this.client.getObjectMDStats(bucketName, bucketInfo, isTransient, this.log, next),
-        ], callback);
+        ], (err, results) => {
+            monitoring.workersCount.inc({ state: err ? 'error' : 'success' });
```

Comment:
this is not a "state", it is a result or status...

Reply:
it is now status -> https://github.com/scality/s3utils/pull/312/commits/094b51c921935a4def339cbd48b61b2f8cf40a3d#diff-612a4c10e405510cdaad879589d4a7d157df6e0ff42b4645d2a21d4d17e41fa9R24

this one gives insights in case of errors which is not really shown in the duration metric